src/css/livesplit.css

```diff
@@ -117,7 +117,7 @@ body {
     background: hsl(0, 0%, 10%);
     padding: 10px;
     padding-top: 15px;
-    width: 250px;
+    width: 260px;
```

Comment:
Is that necessary? The sidebar is already kinda wide and I'd like to keep it as little as possible.

Reply:
Unfortunately, yes. Otherwise you'd end up with the second screenshot.